biogeochem/EDCanopyStructureMod.F90

```diff
@@ -1017,7 +1080,7 @@ subroutine canopy_summarization( nsites, sites, bc_in )
              call carea_allom(currentCohort%dbh,currentCohort%n,sites(s)%spread,&
                   currentCohort%pft,currentCohort%c_area)
 
-             currentCohort%treelai = tree_lai(currentCohort%bl,             &
+             currentCohort%treelai = tree_lai(leaf_c,             &
                   currentCohort%pft, currentCohort%c_area, currentCohort%n, &
```

Comment:
Apologies for this additional naive question but I dont see SLA in this subroutine/function to calculate LAI for each cohort, or is that the c_area?

Reply:
@kovenock updated the LAI calculations to use SLA at the top of the plant, and optionally a higher SLA that increases as one decreases in elevation in the canopy.  But at its heart, the tree_lai() routine should be making use of these parameters.  It is a parameter, not a dynamic quantity, so it does not need to be passed in as an argument.

Reply:
Got it, thanks!

Reply:
@rgknox as far as my comment about bleaf vs leaf_c, here is one of the examples using leaf_c.  Just wanted to make sure whether you had planned for everything to be leaf_c or if in the other case bleaf is appropriate?

Reply:
My take is that we should drop the b_ convention altogether, except we want to describe things in terms of total biomass (including not just carbon, but lignans, water, nutrients, etc).   We do have a subroutine called bleaf(), and I want to change that too.  That subroutine is actually calculating the plants target leaf carbon for a fully flushed, on-allometry, plant of a given size and trimming.  So bleaf(), is not a very concise subroutine name.